gravitee-apim-console-webui/src/components/alerts/alertEnginePromotion/alert-engine-promotion-component.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const AlertEnginePromotionComponent: ng.IComponentOptions = {
```

Comment:
You should create an Angular component instead of an AngularJS one. 
With a call to `downgradeComponent` method in the main module you will then be able to use it in an old component  (like here: https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/management/management.module.ajs.ts#L659)

Do not hesitate to ask @ThibaudAV @gcusnieux @loriepisicchio @jourdiw  for any help on that side ;)

Reply:
Completely agree.

We plan in the future to have new components on AE with this and having a template to help building this will be helpful and this component is a good candidate to start with

https://github.com/gravitee-io/issues/issues/8853

Let's tackle that in another PR if that's ok

Reply:
marking as resolved to enable merging